packages/react-form-renderer/src/condition/condition.js

```diff
@@ -42,6 +52,12 @@ const Condition = React.memo(
       }
     }, [dirty]);
 
+    const setValue = useCallback((setter) => {
+      Object.entries(setter).forEach(([name, value]) => {
+        formOptions.change(name, value);
+      });
+    });
```

Comment:
We still need to keep an empty dependency array here: `[]` otherwise the function will be always re-created.

See [here](https://beta.reactjs.org/apis/react/useCallback#every-time-my-component-renders-usecallback-returns-a-different-function)